R/diagnostics.R

```diff
@@ -2,28 +2,33 @@ library(data.table)
 
 .args <- if (interactive()) {
 	c(
-		file.path("local/output", c("forecast_daily_EC.rds", "forecast_weekly_EC.rds")),
-		file.path("local/output", "diagnostics_EC.csv")
+	    file.path("local/output",
+	              c("forecast_daily_GP.rds",
+	                "forecast_weekly_GP.rds",
+	                "forecast_special_GP.rds"
+	              )
+	    ),
+	    file.path("local/output", "diagnostics_GP.csv")
 	)
 } else {
-	commandArgs(trailingOnly = TRUE)
+    commandArgs(trailingOnly = TRUE)
 }
 
 # Get diagnostic data
-diagnostics_daily_fits <- readRDS(.args[1])$diagnostics |> rbindlist()
-diagnostics_weekly_fits <- readRDS(.args[2])$diagnostics |> rbindlist()
+daily_dt <- readRDS(.args[1])$diagnostics |> rbindlist()
+weekly_dt <- readRDS(.args[2])$diagnostics |> rbindlist()
+special_dt <- readRDS(.args[3])$diagnostics |> rbindlist()
```

Comment:
```suggestion
.args[1:3] |> setNames(c("daily", "weekly", "rescale")) |>
  lapply(readRDS) |> lapply(\(obj) {
    obj$diagnostics |> rbindlist()
}) |> rbindlist(idcol = "type", fill = TRUE)
```

... or something similar. the copy-pasta is a bit annoying.

Reply:
(and can delete the subsequent lines assigning type, binding up)